integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA3336HostHeaderValuePortCheckTestCase.java

```diff
@@ -103,8 +103,12 @@ private void applyProperty(File srcFile, String key, String value) throws IOExce
             fos = new FileOutputStream(destinationFile);
             Properties properties = new Properties();
             properties.load(fis);
+            if (fis != null) {
+                fis.close();
+            }
```

Comment:
Do we need this check since we are releasing this in finally block?

Reply:
We need to release input stream at this point because both input and output streams using the same file. Therefore we cannot access the file in output stream without releasing it in Windows.